scenario_09_sandbox.lua

```diff
@@ -38282,6 +38368,163 @@ function starryChristmas()
 	end)
 end
 
+
+----------------------------------
+--	Custom > One-Offs > Kosai  --
+----------------------------------
+-- Button Text		   FD*	Related Function(s)
+-- -MAIN FROM KOSAI	F	initialGMFunctions
+-- -CUSTOM				F	customButtons
+-- -ONE-OFFS			F	oneOffs
+function kosaiOneOff()
+	clearGMFunctions()
+	addGMFunction("-Main From Kosai",initialGMFunctions)
+	addGMFunction("-Custom",customButtons)
+	addGMFunction("-One-Offs",oneOffs)
+	addGMFunction("Exhuari Riptide warp stabilizer outpost", function()
+		-- find the wormhole close to these coordinates
+		-- we do this to separate one-off logic from terrain setup logic
+		local wh = nil
```

Comment:
Can we name wh something a bit more descriptive, please? Perhaps riptide_to_Icarus_wormhole

Reply:
Actually this one-off (and the other one) is no longer needed, I guess I just pushed it for completeness sake since I used it last session. I'll remove them